scripts/get_changelog_diff.js

```diff
@@ -0,0 +1,26 @@
+/*
+Here we're trying to parse the latest changes from CHANGELOG.md file.
+The changelog looks like this:
+
+## 0.0.3
+- Something #3
+## 0.0.2
+- Something #2
+## 0.0.1
+- Something #1
+
+In this case we're trying to extract "- Something #3" since that's the latest change.
```

Comment:
If I understand correctly from code then this will extract 
```
## 0.0.3
- Something #3
```
, not just `- Something #3`, right?

Reply:
@ffenix113 oh yeah correct.

Reply:
Nice, thank you for confirming.